### ISSUE TITLE
Point figure links to static

### ISSUE THREAD

**Issue by j2kun** [j2kun/math-intersect-programming#9]
Also note the images are mangled a bit

```
2011/06/13/prime-design/

<div class="wp-caption alignleft" id="attachment_89" style="width: 33px">[![](https://i0.wp.com/jeremykun.com/wp-content/uploads/2011/06/23.png?resize=23%2C50 "23")](https://i0.wp.com/jeremykun.com/wp-content/uploads/2011/06/23.png)23
```

**Comment by j2kun** [comment]
Looks like they're already downloaded, but I just need to modify the URLs to point to them.

**Comment by j2kun** [comment]
Fixed in b900a15
